husky-fhir/husky-fhir-emed-ch/husky-fhir-emed-ch-epr/src/main/java/org/projecthusky/fhir/emed/ch/epr/model/emediplan/posology/EMediplanPosology.java

```diff
@@ -0,0 +1,137 @@
+package org.projecthusky.fhir.emed.ch.epr.model.emediplan.posology;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.hl7.fhir.r4.model.OperationOutcome;
+import org.projecthusky.fhir.emed.ch.epr.model.emediplan.EMediplanObject;
+import org.projecthusky.fhir.emed.ch.epr.model.emediplan.enums.CdTyp9;
+import org.projecthusky.fhir.emed.ch.epr.model.emediplan.enums.EMediplanType;
+import org.projecthusky.fhir.emed.ch.epr.model.emediplan.enums.RelativeToMeal;
+import org.projecthusky.fhir.emed.ch.epr.model.emediplan.posology.detail.PosologyDetail;
+import org.projecthusky.fhir.emed.ch.epr.validator.ValidationResult;
+
+import java.time.Instant;
+import java.util.Objects;
+
+/**
+ * Describes when and what amount of medication must be taken.
+ */
+@Data
+public class EMediplanPosology implements EMediplanObject {
+    protected static final boolean DEFAULT_AS_NEEDED_VALUE = false;
+
+    protected static final String DETAIL_FIELD_NAME = "po";
+    protected static final String UNIT_FIELD_NAME = "unit";
+
+    /**
+     * From date. Format: YYYY-MM-DDThh:mm:ss+02:00 or YYYY-MM-DD (ISO 860113 Combined date and time including time zone
+     * or date only) (e.g. 2016-06-16T16:26:15+02:00).
+     */
+    @JsonProperty("dtFrom")
+    @JsonFormat(pattern = "yyyy-MM-dd", timezone = "Europe/Zurich")
```

Comment:
Aren't we losing information here in both ways? The comment says YYYY-MM-DDThh:mm:ss+02:00 is supported.

Reply:
Indeed, I'll fix it.